src/components/Karma/CircleGradient.tsx

```diff
@@ -47,12 +48,16 @@ export const CircleGradient = (props: {
       </LinearGradient>
       {props.fill !== 100 && (
         <Svg
-          height={props.size}
-          width={props.size}
-          style={{position: 'absolute', transform: [{rotateZ: '-90deg'}]}}>
+          height={props.size + Math.ceil(props.width)}
+          width={props.size + Math.ceil(props.width)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The height and width of the SVG have been increased by `Math.ceil(props.width)`. This will make the SVG larger than before. Make sure this change doesn't cause any unexpected layout shifts or overlaps with other UI elements.
```diff
- height={props.size}
- width={props.size}
+ height={props.size + Math.ceil(props.width)}
+ width={props.size + Math.ceil(props.width)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->